app/src/Navigation/index.js

```diff
@@ -1,18 +1,31 @@
 import React, { useContext } from 'react';
+import { AsyncStorage } from 'react-native';
 import BottomTab from './MainNavigation/BottomNavigator';
 import AuthRoutes from './AuthNavigation';
 import { NavigationContainer } from '@react-navigation/native';
 import { UserContext } from '../store/contexts/userContext';
 import Splash from '../pages/Splash';
+import IntroSlide from '../pages/FirstAccessTutorial';
 
 const Routes = () => {
     const { user } = useContext(UserContext);
 
     const isLoadingUserInformation = user && user.showSplash;
     const isUserAuthenticated = user._id;
+
+    const isFinishedIntroSlide = async () => {
```

Comment:
Para que a função tenha o resultado esperado, ela deve se encontrar dentro de um useEffect(). Da forma como a função está sendo implementada ela nunca será chamada, e se for chamada, de alguma forma, será depois que o react renderizou a página, podendo causar um problema de Memory Leak.